rfcs/0000-client-block-filter/0000-client-block-filter.md

```diff
@@ -0,0 +1,136 @@
+---
+Number: "0000"
+Category: Standards Track
+Status: Proposal
+Author: Quake Wang <quake.wang@gmail.com>
+Created: 2022-08-23
+---
+
+# CKB Client Side Block Filter Protocol
+
+## Abstract
+
+This RFC describes a light client protocol allowing clients to obtain compact probabilistic filters of block content from full nodes and download full blocks if the filter matches relevant data.
+
+## Motivation
+
+Light clients allow applications to read relevant transactions from the blockchain without incurring the full cost of downloading and validating all data. Such applications seek to simultaneously minimize the trust in peers and the amount of bandwidth, storage space, and computation required. They achieve this by sampling headers through the fly-client protocol, verifying the proofs of work, and following the longest proof-of-work chain. Light clients then download only the blockchain data relevant to them directly from peers and validate inclusion in the header chain. Though clients do not check the validity of all blocks in the longest proof-of-work chain, they rely on miner incentives for security.
```

Comment:
This paragraph is describing light client protocol not block filter protocol. Would it be better to contribute it to RFC0044?

Reply:
This paragraph is about the background of light client, I think it can be kept in this rfc